### DISCUSSION TITLE
Slim 7+ days "History" widget to put next to the "Checkmark" widget.

### DISCUSSION THREAD

**Discussion by JFMugen** [iSoron/uhabits#1020] [Feature Requests]
![Like this example](https://user-images.githubusercontent.com/61559703/125947712-29b0e86c-c28b-4d44-8bee-fea92d565b8e.png)

Like the example above. Instead of streak, seeing your progress for the last 7 (or more, as long as it is big enough to see) days would be useful to see your progress / manage it easily.

**Comment by VA-MMXXII** [comment]
Interesting idea. It would also be great if the interval of each widget could be set independent of the interval currently selected for that graph inside the app. In other words, if I set up a widget to show the Score on a monthly basis and then set any Score within the app to show quarterly or yearly, the widget interval should remain as I selected it---not as how the app is currently showing that graph.
